### ISSUE TITLE
Errors on build-in packages disabling

### ISSUE THREAD

**Issue by novavision** [BayatGames/SaveGamePro#5]
If I disable some unused build-in packages like Terrain/Terrain Physics/etc I get multiple errors in SaveGame. Would be great if you add package script dependency.

**Comment by hasanbayatme** [comment]
Yeah, these errors arise from Save Game Pro types which are specific scripts for handling serialization of Unity types, so that is where problem comes from, here is a review from one of the customers talking about same subject:

> I also gave up this asset because of the lack of handling for removed Built-In Packages.
> 
> In Unity, You can remove unused Unity built-in packages if you never used them in your game, such as UnityEngine.Terrain. However, if you have removed any packages and you import this asset, problem comes: since SaveGamePro has specific codes for many Unity types (including Terrain of course), while you have removed reference to UnityEngine.Terrain, there will be many errors about missing references in their code and you just have to remove all source files containing errors. The more packages you removed, the more errors you have to handle manually. 
> 
> I hope there are some methods to handle this, automatically - but sadly, there seems to be none in Unity. Unity is the one to blame.

So, the approach would be just to delete those scripts which are related to those removed packages, then you'll be pretty fine.

Actually Save Game Pro is not depended on those packages, those packages previously was integrated inside UnityEngine itself instead of being separated and the user have the permission to disable them, by the way now you can just delete those scripts.

**Comment by novavision** [comment]
Is there no way to check build-in packages?

**Comment by hasanbayatme** [comment]
I didn't check by myself, but it looks no, maybe now they should have added some sort of public methods to the Package Manager window or something like that, but not sure honestly.
